build.sh

```diff
@@ -98,10 +98,12 @@ touch /etc/rc.conf
     echo 'boot_multicons="YES"' >> /mnt/boot/loader.conf
     echo 'boot_serial="YES"' >> /mnt/boot/loader.conf
     echo 'comconsole_speed="115200"' >> /mnt/boot/loader.conf
-    echo 'autoboot_delay="1"' >> /mnt/boot/loader.conf
+    echo 'autoboot_delay="-1"' >> /mnt/boot/loader.conf
```

Comment:
How do you do if you need to recover a broken system and jump back on an older kernel?

Reply:
The docs say: `If set to "-1", no delay will be inserted and loader.conf starts interactive mode only if autoboot has failed.`  
My assumption is that if you need to recover a broken system, you can still access interactive mode.  
Prioritising a kernel for next reboot can be done in upfront via `grub.cfg` iirc.